src/app/feed/info-box/info-box.component.spec.ts

```diff
@@ -22,7 +22,83 @@ describe('Component: InfoBox', () => {
 	});
 
 	it('should create an instance', async(() => {
-		const component = TestBed.createComponent(InfoBoxComponent);
+		const fixture = TestBed.createComponent(InfoBoxComponent);
+		const component = fixture.debugElement.componentInstance;
 		expect(component).toBeTruthy();
 	}));
+
+	it('should give valid result on non-empty ApiResponseResult', async(() => {
+		const fixture = TestBed.createComponent(InfoBoxComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		component.ApiResponseResult = [
+			{
+				hashtags: [
+					'fossasia',
```

Comment:
```FOSSASIA```

Reply:
@simsausaurabh actually I named them after searching `fossasia` on `loklak.org`. Should I still change them?
![screenshot_2019-01-05 loklak search - distributed search for twitter and social media with peer to peer technology](https://user-images.githubusercontent.com/25428397/50727503-b3499080-1141-11e9-9c34-db56fa87dadf.png)

Reply:
Please make hashtags to be ```FOSSASIA```. We follow this in our tweets too.